src/main/java/com/google/sps/servlets/Message.java

```diff
@@ -2,13 +2,13 @@
 import com.google.appengine.api.datastore.Entity;
 
 public class Message {
-    String messageId;
-    String chatroomId;
-    String text;
-    String translatedText;
-    String senderId;
-    String recipientId;
-    Long timestamp;
+    public String messageId;
+    public String chatroomId;
+    public String text;
+    public String translatedText;
+    public String senderId;
+    public String recipientId;
+    public Long timestamp;
```

Comment:
why is is being changed to public

Reply:
So I can access them easily on the servlets without making methods

Reply:
ok. i am not sure it is the best practice in java though. normally variables are made private to a class and accessed only through methods so that 'bad actors' cannot change the value of the variables... but this seems like an easier walkaround